tools/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/tektoncd/pipeline/tools
 
-go 1.18
+go 1.21
+
+toolchain go1.22.3
```

Comment:
Is this change intended?

Reply:
@afrittoli it might not be, but it's doing the same thing automatically on my laptop (go 1.22.3 from nixos) — it automatically adds the `toolchain` entry.